src/ffsplat/conf/format/SOG-canvas.yaml

```diff
@@ -0,0 +1,356 @@
+          index_field: sorted_indices
+
+  - input_fields: [scales]
+    transforms:
+      - simple_quantize:
+          min: 0
+          max: 255
+          dim: 2
+          dtype: uint8
+          round_to_int: true
+
+  - input_fields: [opacities, sorted_indices]
+    transforms:
+      - reindex:
+          src_field: opacities
+          index_field: sorted_indices
+
+  - input_fields: [quaternions, sorted_indices]
+    transforms:
+      - reindex:
+          src_field: quaternions
+          index_field: sorted_indices
+
+  - input_fields: [quaternions] # for tracking decoding_ops
+    transforms:
+      - to_field:
+          to_field_name: quats
+
+  - input_fields: [quats]
+    transforms:
+      - reparametize:
+          method: pack_dynamic
+          to_fields_with_prefix: quaternions_packed_
+          dim: -1
+
+  - input_fields: [quaternions_packed_values]
+    transforms:
+      - remapping:
+          method: scale-sqrt2
+      - remapping:
+          method: "minmax"
+          min: 0
+          max: 255
+
+  - input_fields: [quaternions_packed_values, quaternions_packed_indices]
+    transforms:
+      - combine:
+          method: concat
+          dim: 2
+          to_field: quats
+
+  - input_fields: [quats]
+    transforms:
+      - simple_quantize:
+          min_values: [0, 0, 0, 252]
+          max_values: [255, 255, 255, 255]
+          dim: 2
+          dtype: uint8
+          round_to_int: true
+
+  - input_fields: [sh0, sorted_indices]
+    transforms:
+      - reindex:
+          src_field: sh0
+          index_field: sorted_indices
+
+  - input_fields: [shN, sorted_indices]
+    transforms:
+      - reindex:
+          src_field: shN
+          index_field: sorted_indices
+
+  - input_fields: [opacities]
+    transforms:
+      - reshape:
+          start_dim: 2
+          shape: [1]
+
+  - input_fields: [sh0]
+    transforms:
+      - permute:
+          dims: [0, 1, 3, 2]
+      - flatten:
+          start_dim: 2
+
+  - input_fields: [sh0, opacities] #opacity-sh0 rgba
+    transforms:
+      - combine:
+          method: concat
+          dim: 2
+          to_field: sh0
+
+  - input_fields: [sh0]
+    transforms:
+      - simple_quantize:
+          min: 0
+          max: 255
+          dim: 2
+          dtype: uint8
+          round_to_int: true
+
+  - input_fields: [shN]
+    transforms:
+      - flatten:
+          start_dim: 2
+      - flatten:
+          start_dim: 0
+          end_dim: 1
+      - cluster:
+          method: kmeans
+          num_clusters: 65536 #is dynamic in sogs
+          distance: manhattan
+          to_fields_with_prefix: shN_
+
+  - input_fields: [shN_centroids]
+    transforms:
+      - simple_quantize:
+          min: 0
+          max: 255
+          dim: 2
+          dtype: uint8
+          round_to_int: true
+
+  - input_fields: [shN_centroids]
+    transforms:
+      - sort:
+          method: lexicographic
+          to_field: shN_centroids_indices
```

Comment:
sort: to_field_prefix -> returns forward sorting indices and backward sorting indices -> no argsort method required